src/pt/sinensis/build.gradle

```diff
@@ -1,9 +1,9 @@
 ext {
     extName = 'Sinensis Scan'
     extClass = '.SinensisScan'
-    themePkg = 'madara'
-    baseUrl = 'https://sinensisscan.net'
-    overrideVersionCode = 6
+    themePkg = 'peachscan'
+    baseUrl = 'https://sinensistoon.com'
+    overrideVersionCode = 7
```

Comment:
peachscan version is currently 4, and madara is 35.
35+6 > 4+7; so adjust the version accordingly